### DISCUSSION TITLE
How to return a specific field in an include?

### DISCUSSION THREAD

**Discussion by TheoMer** [prisma/prisma#3498] [Q&A] [answered]
So in Prisma 1 to return a result set and include in it just the id of an address I would do the following:

```ts
const user = await ctx.prisma.query.user(
      {
        where: { email },
      },
      `{ id, name, email, password, address { id },  resetToken, resetTokenExpiry}`
    ).catch(handleSubmitErr);
```

How would I amend the following to achieve the same thing?:

```ts
const user = await prisma.user.findOne({
  where: { email },
  include: { address: true },
})
```

**Comment by ryands17** [comment, marked as answer]
You can use `select` for this

```ts
const user = await prisma.user.findOne({
  where: { email },
  select: { address: { id: true } },
})
```

**Reply by TheoMer** [reply]
@ryands17 Many thanks.

**Reply by TheoMer** [reply]
Actually the following did the trick:

```
            where: { email: email },
            include: { 
              address: { 
                select: {
                  id: true
                } 
              } 
            },
```

**Reply by ryands17** [reply]
Yup. Both will work :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
